### PR TITLE
Portfolio-setup-and-mobile-first project pull request

### PR DESCRIPTION
**Pull request by gitabiy**
In this branch I did:
Add index.html file
Add style.css file
Add .gitignore file
Add linters 
Modified README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <div id="container-three">
+                <img class="ell-15" src="images\Ellipse 15.png" alt="">
+                <div class="fifth">
+                    <img class="ell-4"src="images\Ellipse 4.png" alt="">
+                    <img class="rect-5"src="images\Rectangle 5.png" alt="">
+                </div>
+                <div class="sixth">
+                    <img class="ell-12" src="images\Ellipse 12.png" alt="">
+                    <img class="ell-13" src="images\Ellipse 13.png" alt="">
+                </div>
```
Comment (LionRouge1) [inline index.html:64]: - [ ] Same here too use CSS `background-image`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+.ell-3 {
+  width: 60px;
+  margin-left: 50px;
+  margin-top: -45px;
+}
+
+.introduction {
+  font-size: 65px;
+}
```
Comment (LionRouge1) [inline style.css:73]: - [ ] Please add this CSS property `color: #172B4D` to the title to make it looks like Figma's design as required in the project requirement.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+.profession {
+  font-size: 25px;
+  word-wrap: break-word;
+}
+
```
Comment (LionRouge1) [inline style.css:79]: - [ ] Your paragraph color looks different from Figma's design, kindly fix it.
Screenshot from 2023-03-24 11-12-40

File: style.css [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+#container-two {
+  margin-left: 20px;
+  margin-top: -120px;
+}
+
+@media (min-width: 675px) {
+  body {
+    background-color: beige;
```
Comment (LionRouge1) [inline style.css:92]: - [ ] Please use white color for your website color to follow Figma's design color
Screenshot from 2023-03-24 11-02-31

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @gitabiy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR :heavy_check_mark: 
- Good commit :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @gitabiy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**